docs/components/concepts/document-handling.md

```diff
@@ -0,0 +1,56 @@
+---
+id: document-handling
+title: "Document handling"
+description: "Learn more about integration, secure management, and efficient storage and retrieval of documents across development and production environments."
+keywords: ["document handling"]
+---
+
+Offering a more comprehensive approach to document handling, Camunda now provides efficient storage and retrieval of documents across both development and production environments.
+
+Whether you are working across Connectors, Modeler, or Tasklist, document handling may be beneficial for users who want the following:
+
+- [AWS S3](https://aws.amazon.com/s3/) storage and bucket creation per cluster to securely store and retrieve documents in an external, scalable storage solution, and to ensure storage is properly isolated and managed for each environment.
+- Support for the local file system as a storage option for development and testing purposes to set up and manage documents in local environments.
+- Preview, track, and manage documents associated with process instances.
+
+Below, learn about capabilities of document handling for the related components:
+
+## Modeler
+
+A [form](/components/modeler/forms/camunda-forms-reference.md) can display documents with the [document preview component](/components/modeler/forms/form-element-library/form-element-library-document-preview.md):
+
+![document preview for form](./assets/document-preview.png)
+
+With the [filepicker form element](/components/modeler/forms/form-element-library/forms-element-library-filepicker.md), you may also [upload multiple files at once](/components/modeler/forms/form-element-library/forms-element-library-filepicker.md#configurable-properties). This can be dynamically set using an [expression](/components/modeler/feel/language-guide/feel-expressions-introduction.md).
+
+## Connectors
+
+### Inbound (webhook) Connectors
+
+[Access created documents](/components/connectors/protocol/http-webhook.md) in both the response expression and the result expression, where the `documents` object contains the references for created documents.
+
+![example payload inbound webhook connector](./assets/inbound-webhook-document.png)
+
+### Outbound (REST and IDP-related) Connectors
```

Comment:
@crobbins215 Can you clarify for me which Connectors this will apply to so I can apply the appropriate content to those docs?